source/documents/modify-multiple.txt

```diff
@@ -14,4 +14,146 @@ Modify Multiple Documents
    :depth: 1
    :class: singlecol
 
-You can perform bulk update operations on multiple documents in Compass.
\ No newline at end of file
+You can perform bulk update operations on multiple documents in Compass
+by using the :guilabel:`Update Documents` modal. Performing updates with
+the :guilabel:`Update Documents` modal helps you visualize updates 
+to your data before you apply them.
+
+About this Task
+---------------
+
+- You can use any syntax that works with the ``update`` parameter of
+  :method:`db.collection.updateMany()`. 
+  
+- The :guilabel:`Update Documents` modal does not support any ``options`` 
+  parameters such as upsert, writeConcern, or collation.
+
+Before you Begin
+----------------
+
+The :guilabel:`Update Documents` modal is available starting
+in Compass version ``1.5.X``. For instructions on updating or installing
+the latest Compass version, see :ref:`download-install`. 
+
+Steps
+-----
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Open the bulk update modal
+
+      From the :guilabel:`Documents` tab, click the :icon-lg:`Edit` 
+      :guilabel:`Update` button to display the 
+      :guilabel:`Update Documents` modal. The following 
+      table summarizes the UI of the modal:
+
+      .. list-table::
+         :widths: 40 60
+         :header-rows: 1
+
+         * - UI Element
+           - Description
+
+         * - :guilabel:`Filter`
+           - Any filter criteria specified on the 
+             :guilabel:`Query Bar` applies to the 
+             :guilabel:`Update Documents` modal.
+         * - :guilabel:`Update`
+           - The update syntax that is applied to the 
+             documents specified in the filter criteria. You can use 
+             any syntax that works with the ``update`` parameter of the
+             :method:`db.collection.updateMany()`.
+         * - :guilabel:`Preview`
+           - A preview of documents with the update syntax applied.
+
+   .. step:: Enter the update syntax 
+
+      In the :guilabel:`Update` text field, provide the update 
+      syntax.
+
+      The number of documents affected by the update displays in 
+      the top of the :guilabel:`Update Documents` modal in 
+      parentheses.
+
+      .. note::
+
+         The documents under the :guilabel:`Preview` header show how the 
+         :guilabel:`Update` syntax affects documents in your collection.
```

Comment:
"syntax" makes sense above, because you are talking about the actual syntax that a user has to write. here (and below) it feels a little weird. We could say "Update command" or simply just the bolded "Update" (I may prefer the latter)